### ISSUE TITLE
Accessibility: build success/failure favicons are difficult to distinguish

### ISSUE THREAD

**Issue by nex3** [travis-ci/travis-ci#9392]
I'm pretty sure the favicon Travis uses varies in color based on whether the build failed, succeeded, or is in-progress. But I'm partially colorblind, and it's **really** hard to tell which is which, especially because multiple icons of different colors are rarely close together. It would be really nice if there were some sort of shape-based indicator—a little check, X, or circle—to indicate progress as well as the color of the icon.

**Comment by nex3** [comment]
For reference, GitHub's icons are very accessible: ![github-icons](https://user-images.githubusercontent.com/188/37938379-66221a26-3112-11e8-878f-0f5ce8b2a391.png)

**Comment by nex3** [comment]
This is definitely still an issue.

**Comment by nex3** [comment]
Still relevant.

**Comment by alexishancock** [comment]
Hello, adding to the fray with this one. For the "in Progress" color, I can barely just get what I am reading on the Travis page as the jobs run. For context I ave Tritanopia colorblindness
Example on the Travis Page:
**Normal**
![travis page](https://user-images.githubusercontent.com/4276080/45981720-de80f780-c00a-11e8-85f7-5a1f1621c64b.png)
**Simulated**
![sim_travis](https://user-images.githubusercontent.com/4276080/45981792-1d16b200-c00b-11e8-91e8-b7002f3dd169.png)




I'm requesting that maybe use the yellow that's in use when the job is in progress on the Github side as  a fix since the yellow is deeper.

**Normal:**
![github](https://user-images.githubusercontent.com/4276080/45981850-52230480-c00b-11e8-9123-16d322857efd.png)
**Simulated:**
![sim_travs2](https://user-images.githubusercontent.com/4276080/45981899-75e64a80-c00b-11e8-9e75-1e901aeeead1.png)

**Comment by DrTorte** [comment]
Hello,

Thanks for bringing this up! In regards to the favicon, we looked at it some time ago, though we weren't really able to find a way to make the favicon clearly distinguish between complete and incomplete while still clearly being Mr. T. I'll bring this back up and see if we can take another look. 

In regards to the text colour, though, I'll make sure to bring that up! 

As we're no longer supporting GitHub issues but this issue remains open, I'm creating a copy of this on the forum. [Please see this link](https://travis-ci.community/t/issue-accessibility-build-success-failure-favicons-are-difficult-to-distinguish/1285).

Thanks and happy building!
